convert-hf-to-gguf.py

```diff
@@ -1547,7 +1547,7 @@ def set_vocab(self):
 
     def _reverse_hf_permute(self, weights: Tensor, n_head: int, n_kv_head: int | None = None) -> Tensor:
         if n_kv_head is not None and n_head != n_kv_head:
-            n_head //= n_kv_head
+            n_head = n_kv_head
```

Comment:
This makes `MiniCPMModel._reverse_hf_permute` exactly equivalent to `LlamaModel.permute`. Should `LlamaModel.permute` (which is also a static method) be used instead in `MiniCPMModel`?